### ISSUE TITLE
About Deluxe Chest Keys

### ISSUE THREAD

**Issue by Sacrifice78** [L2J/L2J_DataPack#317]
It's true that in High Five shouldn't drop???

Deluxe Chest Key - Grade 1 --> itemId 6665
Deluxe Chest Key - Grade 2 --> itemId 6666
Deluxe Chest Key - Grade 3 --> itemId 6667
Deluxe Chest Key - Grade 4 --> itemId 6668
Deluxe Chest Key - Grade 5 --> itemId 6669
Deluxe Chest Key - Grade 6 --> itemId 6670
Deluxe Chest Key - Grade 7 --> itemId 6671
Deluxe Chest Key - Grade 8 --> itemId 6672

Can be deleted from droplist if is true?

**Comment by Zealar** [comment]
Yes they is not used anymore and not drop/sell. But remove from drop is not that easy.

**Comment by Sacrifice78** [comment]
YES!, the "groupchances" would change and would have to recalculate it... it is a big big problem!...
